console/src/components/Common/Button/Button.tsx

```diff
@@ -37,8 +40,29 @@ const Button: React.FC<ButtonProps> = props => {
       extendedClassName += 'btn-default';
       break;
   }
+
+  const globals = React.useContext(GlobalContext);
```

Comment:
What about something like `globalsState`?

Reply:
It's referred to as globals everywhere. I think `globals` is okay.